terraform/staging/main.tf

```diff
@@ -122,6 +122,12 @@ module "networking" {
   vpc_flow_cloudwatch_log_expiration_days = 90
 }
 
+module "monitoring" {

```

Comment:
Believe we might need to add this in dev and prod as well? 😊

Reply:
Ah, sorry, knew I'd forgotten to write something on the PR - basically I'm generally just doing stuff for staging, then once the PR has been approved and I know exactly what staging should look like only then do I replicate for dev and prod. Helps to reduce the amount of work I have to do!